packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -270,7 +270,11 @@ class Form<T extends DbObject> extends Component<any,any> {
     });
 
     // run data object through submitForm callbacks
-    data = runCallbacksList({ callbacks: this.submitFormCallbacks, iterator: data, properties: { form: this } });
+    data = runCallbacksList({
```

Comment:
Not sure about why there's a substantial divergence in this file; doesn't seem obviously related to ckeditor, unless it's a Form and requires different behavior?